packages/ove-app-svg/src/client/common/svg.css

```diff
@@ -0,0 +1,30 @@
+.unselectable {
+    -moz-user-select: -moz-none;
+    -khtml-user-select: none;
+    -webkit-user-select: none;
+    -o-user-select: none;
+    user-select: none;
+}
+body {
+    color: #FFF;
+    margin: 0px;
+    overflow: hidden;
+}
+.wrapper, .svg_frame {
+    top: 0;
+    left: 0;
+    border: 0 none;
+    display: block;
+    width: 100%;
+}
+.wrapper {
+    position: fixed;
+}
+.svg_frame {
+    position: absolute;
+    -moz-transform-origin: 0% 0%;
```

Comment:
I  think these "0%" could just be "0'

Reply:
Good question. I did read [worked examples](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin) for this CSS property. It seems that `0 0` should work. However, a better solution could be to have `-moz-transform-origin: top left`. The [spec](https://www.w3.org/TR/css-transforms-1/#transform-origin-property) is unclear as to how `0 0` is treated (apart from initial used value for SVG elements), and `top` and `left` actually maps to `0%`. I will fix this to top/left therefore, see how it works and make the change for all apps that have it.